src/documents/IoCTestingFramework.md

```diff
@@ -122,151 +116,422 @@ fluid.tests.globalCatTest = function (catt) {
+expressing their relative positions in the sequence. `sequenceGrade` can be used together with the existing `sequence` element, but is more regularly
+used without it.
+
+The element `sequenceGrade` holds a string value, designating a component [grade name](ComponentGrades.md) which has been implemented by the
+test case author, descended from the standard framework grade `fluid.test.sequence`. The framework will instantiate a component with this grade as a child component of the
+`testCaseHolder, and then resolve its options path `elements`, whose keys represent namespaces and whose values are of a type `testSequenceElement`, the members of which are described in the following table:
+
+<table>
+    <thead>
+        <tr>
+            <th colspan="3">Members of an <code>testSequenceElement</code> entry within the <code>elements</code> block of a <code>fluid.test.sequence</code> component</th>
+        </tr>
+        <tr>
+            <th>Member</th>
+            <th>Type</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><code>gradeNames</code> (optional)</td>
+            <td><code>{String|Array of String}</code></td>
+            <td>One or more grade names, designating the grades of a component descended from <code>fluid.test.sequenceElement</code></td>
+        </tr>
+        <tr>
+            <td><code>options</code> (optional)</td>
+            <td><code>{Object}</code></td>
+            <td>Any additional options required to construct the <code>fluid.test.sequenceElement</code></td>
+        </tr>
+        <tr>
+            <td><code>priority</code> (optional)</td>
+            <td><code>{Priority}</code> value - see <a href="Priorities.md">Priorities</a> for a full explanation</td>
+            <td>The priority (if any) that the sequence element component should have over any others appearing within the same <code>elements</code> block
+            of its parent <code>fluid.test.sequence</code>. The namespaces used in these priority constraints will be taken from the keys of the <code>elements</code> hash.
+        </tr>
+        <tr>
+            <td><code>namespace</code> (optional)</td>
+            <td><code>{String}</code></td>
+            <td>If present, will override the key of this member of <code>elements</code> in representing the namespace of this element.
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+The members `gradeNames` and `options` in the above table collectively designate an Infusion component derived from `fluid.test.sequenceElement` &mdash; this component will be 
+instantiated as a child component of the `fluid.test.testCaseHolder` and its options member `sequence` will be evaluated. This member `sequence` holds one or more 
+[`fixture elements'](#supported-fixture-records) just as seen in the top-level [`sequence`](#-modules-tests-and-sequence-) element.
```

Comment:
It seems you have used a single quote instead of a backpack after "fixture elements".